getting_started.md

```diff
@@ -21,7 +21,7 @@ source activate [YOUR_CDAT_CONDA_ENV]
 
 Once you've loaded the environment, you should be able to run the examples. They should output a .png file that has the same image as the example.
 
-We strongly recommend using Jupyter notebook for the tutotrials
+We strongly recommend using Jupyter notebook for the tutotrials.  When you type the command below it is best to have navigated to a folder that contains at least one jupyter notebook (file extention .ipynb).
```

Comment:
or folder containing them. Please mention that the start directory will be the top directory you cannot go back higher than this in notebook.